File: src/main/java/com/example/ex3_2_back/entity/Worker.java
package com.example.ex3_2_back.entity;


import jakarta.persistence.*;
import lombok.*;

@Entity
@Builder
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class Worker {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id;
    String name;
    @Builder.Default
    Gender gender = Gender.Unknown;
    String email;
    String phone;
    String address;
    String position;
}

File: src/main/kotlin/com/example/ex3_2_back/entity/Worker.kt
package com.example.ex3_2_back.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id

@Entity
class Worker(
    @Id
    @Column(name = "worker_id")
    var id: Int = 0,
    var gender: Gender = Gender.Unknown
)
